new-pizza.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      body{
+        background-image:url("https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2Fmobile%20yellow.png?v=1604358577851");
+        background-size:100%;
+      }
+    <title>NEW PIZZA</title>
```

Comment:
Title should be outside of the style tag, and styles should be moved to an external file (like new-pizza.css)